themes/app/templates/App/PageType/Layout/NewsPage.ss

```diff
@@ -0,0 +1,25 @@
+<% include PageHeader %>
+<div class="container">
+    <div class="row">
+        <div class="typography col-lg-8">
+            <p>$Created.Format(dd) $Created.Month $Created.Year<p>
+            <p class="lead summary">$Summary</p>
+            <% if $FeatureImage && $IsDisplayed %>
```

Comment:
could swap it this featureImage piece with a more optimised version that Irsan did

```            <% if $FeatureImage && $IsDisplayed %>
                <picture>
                    <source srcset="$FeatureImage.Fill(1920,1080).URL()" media="(min-width: 992px)" />
                    <img src="$FeatureImage.Fill(800,600).URL()" alt="Feature Image" />
                </picture>
            <% end_if %>```